dashboard/src/pages/apps.tsx

```diff
@@ -59,6 +60,20 @@ export default () => {
       return acc
     }, {} as Record<string, Application[]>)
 
+  const countAppsStatus = (state, buildType) => {
+    let num = 0
+    for (let i = 0; i < apps().applications.length; i++) {
+      if (!(apps().applications[i].buildType === buildType)){
+        continue
+      }
+      if (!(apps().applications[i].state === state)) {
+        continue
+      }
+      num += 1
+    }
+    return num
```

Comment:
下のように書いたほうが関数型っぽくてindexの取り違えやnumの足し忘れなどの間違いを防止できます
```suggestion
    return apps().applications.filter((app) => app.state === state && app.buildType === buildType).length
```